tests/queries/0_stateless/00600_replace_running_query.sh

```diff
@@ -37,10 +49,13 @@ $CLICKHOUSE_CURL -sS "$CLICKHOUSE_URL&query_id=42&replace_running_query=1" -d 'S
 
 $CLICKHOUSE_CURL -sS "$CLICKHOUSE_URL" -d "KILL QUERY WHERE query_id = '42' SYNC" > /dev/null
 wait
```

Comment:
This is not enough, because the command can return control to the shell while the query returned result, but not necessary finish running.